AcadeMeData/test_messages.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from AcadeMeData.models import Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.fixture
+def generate_message(user_example, msgID=1, text='This is a test message yo'):
+    user = user_example
+    message = Messages(msgID=msgID, userID=user, text=text)
+    message.save()
+    return message
+
+
+@pytest.mark.django_db
+class TestMessagesModel:
+
+    def test_get_msg(self, generate_message):
+        msg = generate_message
+        msg_test = Messages.get_msg_by_id(1)
```

Comment:
I do not think this method helps in finding messages.
Did you think what are the keys for the messages

Reply:
This method is meant to test the get_msg_by_id function. 
I am initializing a message through a fixture with id=1
then I am inserting that message by id to another variable using the get_msg function.
then asserting if those 2 are the same and asserting if test_msg is a messages instance (object)
How is that not a good test for this specific function?